pkg/bootstrapper/kubelet.go

```diff
@@ -0,0 +1,163 @@
+package bootstrapper
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+// kubeletService struct contains the kubelet specific service information
+type kubeletService struct {
+	// obj is a pointer to the Windows service object
+	obj *mgr.Service
+	// dependents contains a list of services dependent on the current service
+	dependents []*mgr.Service
+}
+
+// newKubeletService creates and returns a new kubeletService object
+func newKubeletService(ksvc *mgr.Service, dependents []*mgr.Service) (*kubeletService, error) {
+	if ksvc == nil {
+		return nil, fmt.Errorf("service object should not be nil")
+	}
+	return &kubeletService{
+		obj:        ksvc,
+		dependents: dependents,
+	}, nil
+}
+
+// getConfig retrieves service config from service object Config()
+func (k *kubeletService) getConfig() (mgr.Config, error) {
```

Comment:
Sorry I should have caught this before. Shouldn't this be just `Config()` as per go naming conventions?

Reply:
done